config/credentials.yml.enc

```diff
@@ -1 +1 @@
-bfQnMCYOHDQjQDD855OdWXAomMjoFCrwDnDIRa6rEesXA9Ee0PseUlA7VS/Y4xmQWYbO4EW6Zn4ViTTUGa1WDTwh0hR06jYsnJV+r7LpGRhnp90v7aM5IZPvtuNsuPFD/gqxcYrk81ivIQ38JG83KTapfEOYl/OYW6CB6SrRXb3N2yZBqTJSBEVNpeBkNSjB/Kiq1edZluD/F30AKA1WkZYQFex1QwjLqcKNiE8kTYF7bpEu901TIhm2jaM/EXi7P3yRnq6QLV9+swxi6KhyHGRGS5kKriUE4bzdQ6CyucNWMtsS5omtGRQmCtEdLIt616bDGg1afCwIXACD5jJelwKYpEMH26I48BCq+3Rpb0SUN1WMC0lajvcC/Iwn7ebGUFPnYeI8FGOXWQJZG3Q1X1laFBJgLtPnJpyFxSfBdjYNBf5lAJfkyiusux3hEp9mCdPdQO6exznDQKJ7PxYYEdxpHCxYWMlOxYc1C/5aH6ZZXWwGtBx0Cy/KVXsSB1H7AXKhHZotW5iGlAuhBCsGiGBkoj+xeIXTcf8abAHjfM2TU/VuFNx7E5DUv6C22hVya51WqoDvGbZHD516a1NzP8yLAUixhuocGT3HI9CgzdlVtOvLKmsxQQl0Dn07XQbc4e52uUVinBzJqLf3OnQY9nKwYRRDHqAZRtOaVa1JLveXcjfIMnXxQSsFdInZp8gzoSK3WcWYKSYy5KbIs0ur7SwGTW6HReEjA9XcPKgIJ+95z4FyAi5JXaft5HndxrQzXSJAFZDN8ICZkF990sOqzcN/mbhcyWv8naSKMaOA/dgxOy+YGuuKg81PMkOw2MB1CC5K7RLuhI8IJI2C4diJrWcC6eu0ybvLhPOXd81MKpEbrS1MJFCHKGgQAR+1aC/90lohvQq4i0HgZKPs1yMKaKRKPdSEQeH/Ov0ZjPJoA7gOUY1BKktfUoTiRxZp3S0cNmNzaZY3hpKtgS36+gpL0LKHbIiUnKxZvaCyWKUUHuc0yUUEkI+QWcNw8IKW9dsz5ESaxJ9lY32kjnPb9tlw1bnFScifUW3x67VYf6ZN7qX/8iioFERMwdek/hvgZow4PSOT97WHEivkUm02F5p3Mok5jVa7m2KShEphsLdSw1tv+gV0T6k0OW1GYzuwjPf3ON/HTOHQRx4jtuBd6Wl8GVpzru/b74jdbKP9su8WIvaiHQDLTJC21YcNB0xKrc1BNEZ+tHlJDMVzRCMGBHxFfNaNw6/iKBGq8dktGzZhvDqKHY86fmWBqb+CLgUsc9usd4m68lNjboVp4xOCgk84G0UXMSmWBkdnJmeMQpgTy62nKZWrE6o6YpCwzVtTugriQne6Z7hgfVFxudBk2gSnz175IbjA6HSuh0t/mWQ4bGuXjES3E295ugf1nk9PlXp7G6VV--uHnKWuOnXuMZGzti--oNNf4mEgz5jY3b1jHYYgCg==
\ No newline at end of file
+F4ql1fRnmopsTXjW19gRA3qWTKyAjqBY5ZDuw96OBxe/nCxHlMT0Q7wZSCH+VSIEKe9QLmx1YUQihyaaRpQolWd4hFcZqDo3IAqUjpLSVONlffE3rKaOEsz0bciPXqjfKUczwbS7XLIdHV6aNwSMmWpJE9qs54mMj1a0NfasItsv2M+8TmS8V04QfiVBI6oE8n5DthhPeBVYQWlMGCYF7dBk1jDAZeuZ3HopHYbxZ7HFIck2IJVLPtwL3y5eYhfYZPE6knAN432jzp7cnlJQ/yBQGivCawcWqGTwGfi23flDlEJO6Kx5y2f+9JFbqbIGShpzcvTcP56n+AC2U+XeCf/YfWbuacfJQ0u48SfylWAIN/qntd9zfBBzPL+3tYTdd5eX1z/6OZGfDYIlBTnjDHGbiLf6eGni3fvCOn7R/6rrI4GSuJY5BkuM1iCvX5grGj1nJSfcNiz79EX0DlgrQ8gD3mA6pHfq8WyddBd9ue3N+MJTK0DvjnRf6q6x+HILOUF3TP5yhfK7uO1Nx0hBYzcMDoV6HWmOaxF3AI62ipRccNFTDF00cXcZDZYBNf+lths+57DUufB618hg1XXpm14HZsprnuvH+7/qlZ5pmZUv5nGIp/w+KIViUSLpgvkdNa3vywTttILvLMaQoo95brnX/vHy8LiOUmx5Dhw3b5VNy/pJBHi/YPK/mzCvbVrOT6UIMgwWgOwnj30xw5B9gYvd3VZG4QXkSJL7pTN5qp2vioHE7Q7zRTERxxZ61G8WFjFhANJgYn6Ma6zrBx6MwN86tyJ0q8RSMacqI+pPXQ0HXMeOK+pwvr6mo7GlD5zeWpddFbJsncXO9kjfA8YQI5ZkejVnRdBKOejBTHK1CeP9AO8J7z7YXGiogofp6jbrdecykNTqXYlSwQuHo+EUcQ8aMuiGojXRZtVW+2DNoeR3Ms5lzuTU+kKj/veLNHqTPmIg79Jv0FSO9ylz+UkuAzeoJlfS3KwJyhwTMR4IJSVqOgwe8C+xEe0ywP4B0O0+txBUsg0yQOzOlKph2ZQ/zJqpSF0UMgv+3UJsAdIZ/tYrkoEt9JqqQwEU8LfSrwE/EIaFAsTHPbEhZJZZYllsxSzLoHxVAJQk6MCwLWsRXHkfvg01MT1BfAGUnv6crjvcLonA+uSoHxijwCbxMdtokoYRHx6/vIgOFywWiRTMgXrOKfB1jJyPIa0HF7Zk+3B04RtGY+GwJhugMxYAOg3b4VqCE/IG2rEQ9Qf/CwuO1PAHWVpSgp0Hk2TqLcx7nm2DZ7rJVrJX6xacWfGmjhrPkI1fzfW6gS9R90dDaQqptrqjMeOpI30zUYHxuMt1reM7b603m36zGlXJfivH4D80--c6lnZYpOeKkI4IeO--F9Y88xORPJx7BUffPAfaog==
```

Comment:
Did anything actually change here? If not, please revert this file and get the development key from @kbiala.

Reply:
I'm not sure if there should be ports after solana testnet urls? there's separate variable rpc_port and for me it worked when I removed the ports in links.

Reply:
Your tests should pass with the current credentials file -- no changes required for this story. Do you also have `config/cluster.yml`?